src/utils/commonValidationSchemas.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
if there is any usage of `any`, lets rather disable locally, not for the whole file